### PR TITLE
Social Authentication challenges added to Advanced Node/Express section

### PR DESCRIPTION
**Pull request by JosephLivengood**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Checklist:

- [x] Tested changes locally.

#### Description

Batch 2 of the Advanced Node and Express section to get testing started asap

### CODE REVIEW

File: seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json [hunk 1]
```diff
@@ -390,6 +390,117 @@
...
       "translations": {}
-    }  
+    },  
+    {
+      "id": "589a69f5f9fc0f352b528e6f",
+      "title": "Social Authentication with OAuth",
+      "description": [
+        [
+          "",
+          "",
```
Comment (Bouncey) [inline seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json]: Why are you inserting two blank lines into the instructions?

Would a `<hr>` be better?

Comment (JosephLivengood) [inline seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json]: I believe those are fields for an Image being a waypoint

Comment (Greenheart) [inline seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json]: I think `challengeType` 7 is the new type used for the introduction- type of challenges. Their descriptions consist of arrays of 4 string, each having a different meaning like images for example.

Comment (JosephLivengood) [inline seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json]: Correct- last one being a requiered link :+1:

Comment (Bouncey) [inline seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json]: `challengeType` 7 tells react it is a `step` challenge and to use the `<Step /> component as the view. This is what gives the intro page a `next` button.

OK, so they are placeholders, not a problem.

**Comment by QuincyLarson** [comment]
@JosephLivengood excellent work. @Bouncey thanks for helping QA!

**Pull request merged**